lib/Sema/InstrumenterSupport.cpp

```diff
@@ -0,0 +1,28 @@
+//===--- InstrumenterSupport.h - Instrumenter Support ---------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+//  This file implements the supporting functions for writing instrumenters of
+//  the Swift AST.
+//
+//===----------------------------------------------------------------------===//
+
+#include "InstrumenterSupport.h"
+
+using namespace swift;
+
+namespace swift {
+namespace instrumenter_support {
```

Comment:
Nitpick: We don't generally reopen namespaces within .cpp files. Instead, just add another `using namespace`, or fully qualify the method name.